routes/api.php

```diff
@@ -90,3 +90,14 @@
 
 Route::post('/settings/privacy/toggle-previous-tags', 'ApiSettingsController@togglePreviousTags')
     ->middleware('auth:api');
+
+// Teams
+Route::prefix('/teams')->group(function () {
+    Route::get('/list', 'API\TeamsController@list');
+    Route::post('/create', 'API\TeamsController@create');
+    Route::patch('/update/{team}', 'API\TeamsController@update');
+    Route::post('/join', 'API\TeamsController@join');
+    Route::post('/leave', 'API\TeamsController@leave');
+    Route::get('/types', 'API\TeamsController@types');
+    Route::get('/leaderboard', 'Teams\TeamsLeaderboardController@index');
```

Comment:
Can we group the get, post, and other requests together please, and list them A-Z

Reply:
I've listed them by relevance/importance, but yeah, we can do that 👍